demo/index.html

```diff
@@ -9,6 +9,8 @@
     gtag('config', 'UA-970836-25');
   </script>
   <title>React Components / Zendesk Garden</title>
+  <meta name="application-name" content="Zendesk Garden">
+  <meta name="description" content="Garden is a design system for Zendesk where we grow beautifully simple and accessible UI components.">
   <link href="//zendeskgarden.github.io/css-components/bedrock/index.css" rel="stylesheet">
```

Comment:
Can we also provide a `keywords` meta? Can probably steal the list from our package.json. `"components", "garden", "react", "zendesk"`